server/templates/pages/src/index.html

```diff
@@ -32,7 +32,7 @@
           <!-- endbuild -->
         <![endif]-->
     </head>
-    <body>
+    <body{{#if flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/if}}>
```

Comment:
> I cannot get this to work, the flowId is always undefined, does it work for you?

Yep, works fine for me:

<img width="690" alt="screen shot 2016-03-07 at 12 36 37" src="https://cloud.githubusercontent.com/assets/64367/13569932/069e65ee-e463-11e5-9cd6-49bf1a3406f0.png">

Forgive me for asking stupid/obvious questions, but:

* Did you restart the server?
* Did you hit the server with a fresh request (rather than navigating within the client app I mean)?